staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4264,6 +4264,36 @@ type PodResourceClaimStatus struct {
 	ResourceClaimName *string `json:"resourceClaimName,omitempty" protobuf:"bytes,2,opt,name=resourceClaimName"`
 }
 
+// PodExtendedResourceClaimStatus is stored in the PodStatus for each extended
+// resource requests backed by DRA. It stores the generated name for
+// the corresponding special ResourceClaim created by scheduler.
+type PodExtendedResourceClaimStatus struct {
+	// RequestMapping identifies the mapping of <container, extended resource backed by DRA> to  device request.
+	// +patchMergeKey=requestName
+	// +patchStrategy=merge,retainKeys
+	// +listType=map
+	// +listMapKey=requestName
```

Comment:
Isn't `ExtendedResourceName` also a key?
```suggestion
	// +listMapKey=requestName
	// +listMapKey=extendedResourceName
```

Reply:
requestName has 1:1 mapping to the requestMapping, but extendedResourceName is 1:N mapping to the requestMapping, <ontainerName, extendedResourceName? has 1:1 mapping to the requestMapping though.

Reply:
removed listMapKey